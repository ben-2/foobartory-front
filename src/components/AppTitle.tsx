import React, { useState } from 'react';

import styles from './AppTitle.module.css';

export function AppTitle() {
  return (
    <>
      FooBartory
    </>
  );
}
