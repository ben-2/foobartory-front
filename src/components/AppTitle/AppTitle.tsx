import styles from './AppTitle.module.css';

export function AppTitle() {
  return (
    <>
      <div className={styles.title}>
        FooBartory
      </div>
    </>
  );
}
