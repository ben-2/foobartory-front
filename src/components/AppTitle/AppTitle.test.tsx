import { AppTitle } from './AppTitle';
import { shallow } from 'enzyme';

describe('<AppTitle />', () => {
  it('renders Foobartory title', () => {
    const wrapper = shallow(<AppTitle />)
    expect(wrapper).toMatchSnapshot();
  })
});
