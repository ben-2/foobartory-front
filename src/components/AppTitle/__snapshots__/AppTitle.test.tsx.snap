// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppTitle /> renders Foobartory title 1`] = `ShallowWrapper {}`;
